spec/features/grading/staff/evaluating_submissions_spec.rb

```diff
@@ -55,6 +58,17 @@
     end
   end
 
+  describe 'global flag' do
+    scenario 'global rating group has global flag', js: true do
+      visit submission_evaluation_path(submission)
+
+      within "#evaluation-group-#{global_evaluation_group.id}" do
+        expect(page).to have_css('fi-flag')
```

Comment:
The `have_css` matcher expects a css selector. 

```suggestion
        expect(page).to have_css('.fi-flag')
```